firmware-management/updated/updated.cpp

```diff
@@ -11,11 +11,14 @@
  * When a request is received, UpdateD will hand off responsibility for applying
  * the update to swupdate. */
 
+#include <unistd.h>
+
 int main()
 {
     while(1)
```

Comment:
In https://github.com/ARMmbed/meta-mbl/pull/812 @diego-sueiro and I have suggested that we do a `fork` in this code rather than relying on systemd to do it for us, so can we add some forking/daemonizing code please? `daemon` from unistd.h might help.

Reply:
In ARMmbed/meta-mbl#812 we decided to let systemd do the forking, so resolving.